config/locales/server.en.yml

```diff
@@ -2751,6 +2754,7 @@ en:
 
     errors:
       invalid_css_color: "Invalid color. Enter a color name or hex value."
+      invalid_datetime: "Invalid datetime format. Use ISO 8601 format (e.g., 2024-12-29T15:30:00Z)."
```

Comment:
Will people ever be able to set the datetime manually or will it always be via the input? If that's the latter, that error message could just be "invalid datetime, please ensure you've entered a valid date & time" ?